Libraries/Text/RCTTextKeyValueConstants.h

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+extern NSString *const kNewlineRawValue;
```

Comment:
Please use RCT prefix for constants, and don't prefix with k.

Also, I'm not sure what the point of having these constants is, vs sending the actual ascii values of those keys?